packages/eslint-plugin/src/rules/indent.ts

```diff
@@ -317,7 +320,7 @@ export default createRule<Options, MessageIds>({
         });
       },
 
-      TSIndexedAccessType(node: TSESTree.TSIndexedAccessType) {
+      TSIndexedAccessType(node: TSESTree.TSIndexedAccessType): void {
```

Comment:
`@typescript-eslint/explicit-function-return-type` began reporting on these once we stopped using `Object.assign`.